library/alloc/src/vec.rs

```diff
@@ -201,6 +201,26 @@ use crate::raw_vec::RawVec;
 /// can be slow. For this reason, it is recommended to use [`Vec::with_capacity`]
 /// whenever possible to specify how big the vector is expected to get.
 ///
+/// A vector containing the elements `'a'` and `'b'` with capacity 4 can be
```

Comment:
The comments on the other PR wrt the formulation here has not been addressed. It would be helpful to the casual reader to make it explicit that the visualization is for *the raw representation of the `Vec` object that lives on the stack* which contains a pointer to the two elements *on the heap*

Reply:
It is not the pointer to two elements, it is the pointer to the first element, no?

Reply:
indeed. Maybe "a pointer to an allocation containing the two elements" is better?

Reply:
Not better, I don't think it is still accurate enough as pointer does not know there are two elements.